deployment/docker/get_addres/getaddress.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Number of addrs to process, default to 3 if not specified
+num_addrs=${1:-1}
```

Comment:
Doesn't match the comment. Default is `1`.

Reply:
God bless copy paste